l10n_co_geo/data/res_bank.xml

```diff
@@ -0,0 +1,864 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+  <data noupdate="1">
+    <!-- <record model="res.bank" id="bank0">
```

Comment:
I left them because I'm not sure if they are actually relevant banks or not. I would like to leave them there to start discussion among contributors and not loose the base set of information. Is this goal achievable?